CommonMark.Tests/TableTests.cs

```diff
@@ -0,0 +1,342 @@
+﻿using CommonMark.Syntax;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+
+namespace CommonMark.Tests
+{
+    [TestClass]
+    public class TableTests
+    {
+        static CommonMarkSettings ReadSettings;
+        static CommonMarkSettings WriteSettings;
+
+        static TableTests()
+        {
+            ReadSettings = CommonMarkSettings.Default.Clone();
+            ReadSettings.AdditionalFeatures = CommonMarkAdditionalFeatures.GithubStyleTables;
+            ReadSettings.TrackSourcePosition = true;
+
+            WriteSettings = CommonMarkSettings.Default.Clone();
+            WriteSettings.AdditionalFeatures = CommonMarkAdditionalFeatures.GithubStyleTables;
+        }
+
+        [TestMethod]
+        public void SimpleTable()
+        {
+            var markdown = "First Header | Second Header\n------------- | -------------\nContent Cell | Content Cell\nContent Cell | Content Cell\n";
+
+            var ast =
+                CommonMarkConverter.Parse(
+                    markdown,
+                    ReadSettings
+                );
+
+            string html;
+            using (var str = new StringWriter())
+            {
+                CommonMarkConverter.ProcessStage3(ast, str, WriteSettings);
+                html = str.ToString();
+            }
+            Assert.AreEqual("<table><thead><tr><th>First Header</th><th>Second Header</th></tr></thead><tbody><tr><td>Content Cell</td><td>Content Cell</td></tr><tr><td>Content Cell</td><td>Content Cell</td></tr></tbody></table>", html);
```

Comment:
This is according to the GHFM-specs, i think its better to leave it as is to have it the same as the spec.